diff --git a/project.clj b/project.clj
index 3195d7b..e93e812 100644
--- a/project.clj
+++ b/project.clj
@@ -24,7 +24,7 @@
                                     :username :env/clojars_username
                                     :password :env/clojars_password
                                     :sign-releases false}]]
-  :plugins [[thomasa/mranderson "0.5.3"]]
+  :plugins [[thomasa/mranderson "0.5.4-SNAPSHOT"]]
   :mranderson {:project-prefix  "refactor-nrepl.inlined-deps"
                :expositions     [[org.clojure/tools.analyzer.jvm org.clojure/tools.analyzer]]
                :unresolved-tree false}
